### PR TITLE
Blog promoting Prometheus at CloudNativeCon.

### PR DESCRIPTION
**Pull request by tomwilkie**
@Gouthamve @brancz WDYT?

### CODE REVIEW

File: content/blog/2017-11-30-prometheus-at-cloudnativecon.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+uses Prometheus for monitoring, and open-source tools they have
+built to implement continuous delivery.  Ilya will go on to explain step-by-step
+how simple it is to instrument an app, using a very generic Node.js app as reference.
+
+## ["If you Don’t Monitor your Infrastructure, you Don’t Own it!” Regain Control Thanks to Prometheus](https://kccncna17.sched.com/event/CU5w/if-you-dont-monitor-your-infrastructure-you-dont-own-it-regain-control-thanks-to-prometheus-i-etienne-coutaud-guillaume-lefevre-octo-technology) (4:25pm - 5:00pm)
+
+The French FedEx company uses Prometheus to monitor their infrastructure.  This talk
+will discuss the use of Prometheus in production, why they chose Prometheus, how they
+integrated it, configured it and what kind of insights they extracted from the
+whole infrastructure.  In addition, this talk with discuss how Prometheus changed
```
Comment (jamtur01) [inline content/blog/2017-11-30-prometheus-at-cloudnativecon.md]: will discuss

File: content/blog/2017-11-30-prometheus-at-cloudnativecon.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+their way of working, how they implemented self-healing based on Prometheus, how they
+configured systemd to trigger AlertManager API, integration with slack and other cool stuff.
+
+## [Prometheus Happy Hour](http://kubecon.freshtracks.io/) (8-10pm)
+
+After such a busy day, we'll be off to Westin rooftop at Azul to soak up the
```
Comment (jamtur01) [inline content/blog/2017-11-30-prometheus-at-cloudnativecon.md]: off to the

File: content/blog/2017-11-30-prometheus-at-cloudnativecon.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+Austin skyline, drink in hand!  Go and [reserve you spot now](http://kubecon.freshtracks.io/).
```
Comment (jamtur01) [inline content/blog/2017-11-30-prometheus-at-cloudnativecon.md]: your spot

**Comment by tomwilkie** [comment]
Thanks @jamtur01!

File: content/blog/2017-11-30-prometheus-at-cloudnativecon.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+include hands-on access to a live Prometheus and Kubernetes cluster, allowing you
+to experiment with PromQL queries to gain deeper insights into your Kubernetes clusters.
+
+## [The RED Method: How To Instrument Your Services](https://kccncna17.sched.com/event/CU8K/the-red-method-how-to-instrument-your-services-b-tom-wilkie-kausal) (2:45pm - 3:20pm)
+
+Tom from Kausal will cover patterns of application instrumentation, where and when they are applicable, and how they can be implemented with Prometheus.  Covering Google’s Four Golden Signals, the RED Method and the USE Method, this talk will also discuss why consistency is an important approach for reducing cognitive load. Finally it will discuss the limitations of these approaches and what can be done to overcome them.
+
+## Meet the Maintainers (3:30pm)
+
+Prometheus developers Frederic Branczyk, Goutham Veeramachaneni and Tom Wilkie will
```
Comment (brancz) [inline content/blog/2017-11-30-prometheus-at-cloudnativecon.md]: Most people know us by either our twitter or github profile, maybe worth linking out to.

**Pull request merged**